Plugins/Santa/SantaPlugin.py

```diff
@@ -0,0 +1,177 @@
+from skpy import SkypeNewMessageEvent, SkypeEditMessageEvent
+from numpy import random
```

Comment:
Using `numpy` for `random` is overkill, please replace it with usage of built-in library.